mateo-mustapic/assignments/liferay-app/sdk/portlets/todo-portlet/docroot/css/main.css

```diff
@@ -72,21 +100,43 @@
 	}
 
 	@media screen and (max-width: 600px) {
-		.counter {
-			font-weight: bold;
-		}
-
 		.list-container h2 {
 			text-align: center;
 		}
 
 		.task-list {
-			margin-right: 3%;
+			margin-right: 5%;
+		}
+
+		.finished-task {
+			width: 95%;
 		}
 
-		.task-list li, .add-item, .input-item {
+		#_todo_WAR_todoportlet_task {
 			border-radius: 0;
+			width: 97%;
+		}
+
+		.task-list li, #_todo_WAR_todoportlet_add {
 			width: 100%;
 		}
+
+		.task-list li, #_todo_WAR_todoportlet_add, .finished-task {
+			border-radius: 0;
+		}
+
+		#_todo_WAR_todoportlet_add {
+			margin: 0;
+		}
+
+		.delete-task {
+			border-radius: 0;
+		}
+
+		span {
```

Comment:
Why are you reverting my source format?

Don't use base html elements (span) for styling of one specific element.